<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Accelerometer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,2">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>设备及通道</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>设备A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>设备B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>设备C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>设备D</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>传感器通道</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3">
            <item>
             <property name="text">
              <string>通道A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>通道B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>通道C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>通道D</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_0">
            <property name="font">
             <font>
              <family>阿里巴巴普惠体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>参数1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_1">
            <property name="font">
             <font>
              <family>阿里巴巴普惠体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>参数2</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="font">
             <font>
              <family>阿里巴巴普惠体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>参数3</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="font">
             <font>
              <family>阿里巴巴普惠体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>参数4</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>输出参数</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>图像频率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>20Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2kHz</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>保存频率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_4">
            <item>
             <property name="text">
              <string>20Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2kHz</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>输出文件</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>.csv</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Reserved</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>归零</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start_pause">
           <property name="text">
            <string>开始接收</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_net">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>网络状态</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>图像区</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>550</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>550</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>图像操作</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
            <item>
             <widget class="QLabel" name="label_time">
              <property name="text">
               <string>时间尺度：1000ms</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_voltage">
              <property name="text">
               <string>幅值尺度：100mV</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="time_minus_btn">
                <property name="text">
                 <string>缩小</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="time_plus_btn">
                <property name="text">
                 <string>增大</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_wave">
                <property name="text">
                 <string>▶</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="voltage_minus_btn">
                <property name="text">
                 <string>缩小</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="voltage_plus_btn">
                <property name="text">
                 <string>增大</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
